drivers/can/can_mcp2515.c

```diff
@@ -361,11 +361,8 @@ static int mcp2515_set_timing(const struct device *dev,
 	/* CNF1; SJW<7:6> | BRP<5:0> */
 	__ASSERT(timing->prescaler > 0, "Prescaler should be bigger than zero");
 	uint8_t brp = timing->prescaler - 1;
-	if (timing->sjw != CAN_SJW_NO_CHANGE) {
-		dev_data->sjw = (timing->sjw - 1) << 6;
-	}
-
-	uint8_t cnf1 = dev_data->sjw | brp;
```

Comment:
As far as I can see, the `sjw` parameter is now unused and can be removed from `struct mcp2515_data` in `can_mcp2515.h`.

Reply:
Thanks, fixed.